src/styles/UserProfilePage.css

```diff
@@ -62,6 +106,13 @@
 
 .user-profile-general-form-header-buttons button {
```

Comment:
probably in best interest to have a common button component instead of making one from scratch every time @phoebeli23 to be used in other pages (primary + secondary)